src/main/java/seedu/partyplanet/logic/parser/ParserUtil.java

```diff
@@ -135,15 +135,15 @@ public static Email parseEmail(String email) throws ParseException {
     public static Birthday parseBirthday(String birthday) throws ParseException {
         requireNonNull(birthday);
         String trimmedBirthday = birthday.trim();
-        if (birthday == Birthday.EMPTY_BIRTHDAY_STRING) {
+        if (birthday.equals(Birthday.EMPTY_BIRTHDAY_STRING)) {
```

Comment:
I believe the `==` check here was intentional cos it is checking the exact object (the static string).
Otherwise LGTM

Reply:
Sounds good, it should be more efficient that way. Will make the change.